frontend/app/views/whatson/calendarList.scala.html

```diff
@@ -0,0 +1,43 @@
+@(groupedEvents: scala.collection.SortedMap[org.joda.time.LocalDate, scala.collection.SortedMap[org.joda.time.LocalDate, Seq[model.RichEvent.RichEvent]]], pageInfo: model.PageInfo)
```

Comment:
This gets pretty crazy when nested like this, I'm sure there's a terser way of modelling this, cc/ @rtyley